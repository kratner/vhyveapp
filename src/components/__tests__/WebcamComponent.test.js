import React from "react";
import { shallow } from "enzyme";
import WebcamComponent from "../WebcamComponent";

describe("WebcamComponent", () => {
  it("renders correctly", () => {
    const wrapper = shallow(<WebcamComponent />);
    expect(wrapper).toMatchSnapshot();
  });
});
