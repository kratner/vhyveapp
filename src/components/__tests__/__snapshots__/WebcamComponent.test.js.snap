// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`WebcamComponent renders correctly 1`] = `ShallowWrapper {}`;
